### ISSUE TITLE
Cannot find the module cirtorch 

### ISSUE THREAD

**Issue by chakrabortyrajatsubhra** [filipradenovic/cnnimageretrieval-pytorch#66]
trying to run python3 -m cirtorch.examples.train YOUR_EXPORT_DIR --gpu-id '0' --training-dataset 'retrieval-SfM-120k' 
            --test-datasets 'roxford5k,rparis6k' --arch 'resnet101' --pool 'gem' --loss 'contrastive' 
            --loss-margin 0.85 --optimizer 'adam' --lr 5e-7 --neg-num 5 --query-size=2000 
            --pool-size=22000 --batch-size 5 --image-size 362 but always saying cannot find module cirtorch.

**Comment by filipradenovic** [comment]
Take a look at Issue #51.
